regex/mx.regex/suite.py

```diff
@@ -44,7 +44,7 @@
   "name" : "regex",
 
   "version" : "21.3.3.1",
-  "release" : False,
+  "release" : True,
```

Comment:
I would avoid this kind of change. We can either do it in the merge commit or in a following commit.

Reply:
Fair enough.

Reply:
@zakkak On second thought, is it worth doing? The upstream branch has it set to true:
https://github.com/oracle/graal/blob/release/graal-vm/21.3/regex/mx.regex/suite.py#L47

 Could we do a version bump to `21.3.4.0` instead and set it back to false in a separate commit? Would that work for you?

Reply:
Yes, that's what I was referring to in "or in a following commit." :)

Reply:
NVM. I've re-done the merge and let it to `False` now.